transformers-compat.cabal

```diff
@@ -52,23 +52,28 @@ source-repository head
 flag two
   default: False
   description: Use transformers 0.2. This will be selected by cabal picking the appropriate version.
-  manual: True
+  manual: False
 
 flag three
   default: False
-  manual: True
+  manual: False
   description: Use transformers 0.3. This will be selected by cabal picking the appropriate version.
 
 flag four
   default: False
-  manual: True
+  manual: False
   description: Use transformers 0.4. This will be selected by cabal picking the appropriate version.
 
 flag five
   default: False
-  manual: True
+  manual: False
   description: Use transformers 0.5 up until (but not including) 0.5.3. This will be selected by cabal picking the appropriate version.
 
+flag five-three
```

Comment:
Why do we need a dedicated flag for this?

Reply:
For symmetry